### DISCUSSION TITLE
Adding keycast to NonGNU ELPA

### DISCUSSION THREAD

**Discussion by phikal** [tarsius/keycast#19] [Support] [answered]
Hi, I was wondering if you were interested in adding keycast to NonGNU ELPA. I remember that you prefer adding packages yourself, so I just wanted to make sure.

**Comment by tarsius** [comment, marked as answer]
Maybe I will eventually add most of my packages to [Non]GNU Elpa, but it is a very very low priority. I also don't want you or anybody else to add this or any of my other packages to NonGNU Elpa at this time.

The act of adding the corresponding entries to elpa-packages only takes a few minutes and sending a message to emacs-devel also doesn't take much longer. If I wanted to do it at this time and if I believed doing this would truly only involve these steps, then I would already have done it.

So please stop offering to add my packages resp. suggesting that I do it. I feel pressured and would much rather concentrate on more important things than to give in and spend time on this, mainly to get it over with.

For example, there are a few packages that I actually want to add soonish, because in those cases I feel there is real value in doing so. Adding other packages first, causes delays in adding the packages that I actually want to add.

**Reply by tarsius** [reply]
This (and especially earlier iterations) was a weird response by me. That sometimes happens when I am stressed and am asked to do something I don't want to do for one reason or another, and forget about my resolution to not immediately respond with an emotional brain dump. So anyway...

I saw your talk and think it is great you are doing that work.

I plan to add my packages over time, but I am currently focused elsewhere and would prefer if you didn't ask me to add certain packages to NonGNU Elpa now. That's about I should have replied to your request. Sorry for the outburst you got instead, it's not your fault I am a bit burned out right now.

**Reply by phikal** [reply]
No problem, I didn't assume any malice. If there is anything I can help with, just say so and I'll do what I can :).

**Reply by tarsius** [reply]
Okay, this package is ready for addition to NonGNU Elpa. I've added the `Package-Version` header. Could you handle the rest please?

**Reply by phikal** [reply]
Done, there seem to have been no issues. Thank you for preparing everything on your end! If you are interested, a badge for keycast will be avalaibe under https://elpa.nongnu.org/nongnu/keycast.svg as soon as the archive is rebuilt.

**Reply by tarsius** [reply]
Thanks!

> If you are interested, a badge ...

For packages that already feature other badges I intend to add the [non]gnu badges as well, but actually I prefer to no add any at all. They remind me too much of those silly "best viewed with netscape/internet explorer" gifs. [I might eventually come around to see some benefit in them, after all they link to relevant pages. Lets give it some time. :grin: ]

**Reply by phikal** [reply]
FIWI I agree. Using images in README files often can make them harder to read offline, but some people really like using a git repository as if it were a website :/
